### ISSUE TITLE
运行basic_demo报错：The attention mask is not set and cannot be inferred from input

### ISSUE THREAD

**Issue by WENBO-Z-H** [THUDM/GLM-4#395]
### System Info / 系統信息

Ubuntu 20.04
Python 3.12  
CUDA 11.6
torch 2.4.0
transformers 4.42.4

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

您好，我成功根据demo_basic的requirements安装了依赖，包括transformers 4.42.4，但在运行原始trans_cli_demo.py文件后(仅修改了模型路径)，提问得到的回答如下：
You: hello
GLM-4:The attention mask is not set and cannot be inferred from input because pad token is same as eos token.As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
请问应该怎么解决呢？

### Expected behavior / 期待表现

希望得知产生问题的原因和解决方法

**Comment by zRzRzRzRzRzRzR** [comment]
这个是正常现象吧，应该是能正常回答的，这个警告可以不管

**Comment by WENBO-Z-H** [comment]
您好，似乎不能正常回答，弹出这个警告后再过十几秒会报错，我刚开始接触这方面的技术，还不太会排查
![image](https://github.com/user-attachments/assets/6d323300-f5cb-4d4b-9f91-8c989312e0cd)

**Comment by zRzRzRzRzRzRzR** [comment]
这个应该是你cuda的问题吧？
你给个你的环境信息，cuda版本
torch transformers等

**Comment by WENBO-Z-H** [comment]
您好，我按照建议试了一下，确实是cuda的问题。之前的11.8还是低了，升级到12.3就好了，感谢感谢

**Comment by 307yin** [comment]
我12.1还是这个问题
